testproject/requirements.txt

```diff
@@ -19,3 +19,4 @@ coverage==5.5
 
 isort
 flake8
+flaky
```

Comment:
what is that for? I mean, I know what's that library but you don't use it anywhere in PR

Reply:
apologies, side effect of setting up the project 👍